<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZenSudoku</class>
 <widget class="QMainWindow" name="ZenSudoku">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ZenSudoku</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(../MyResources/5.png)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>1920</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string notr="true"/>
      </property>
      <property name="styleSheet">
       <string notr="true">background: none;</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <widget class="QWidget" name="Main">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="QPushButton" name="newGame_1">
        <property name="geometry">
         <rect>
          <x>1170</x>
          <y>170</y>
          <width>501</width>
          <height>161</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Franklin Gothic Demi Cond</family>
          <pointsize>32</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 20px;
border-width: 2px;
border-color: rgb(0, 0, 0);
border-style: solid;
background: transparent
</string>
        </property>
        <property name="text">
         <string>New Game</string>
        </property>
       </widget>
       <widget class="QPushButton" name="loadGame">
        <property name="geometry">
         <rect>
          <x>1170</x>
          <y>430</y>
          <width>501</width>
          <height>161</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Franklin Gothic Demi Cond</family>
          <pointsize>32</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 20px;
border-width: 2px;
border-color: rgb(0, 0, 0);
border-style: solid;
background: transparent
</string>
        </property>
        <property name="text">
         <string>Load Game</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Quit_2">
        <property name="geometry">
         <rect>
          <x>1170</x>
          <y>680</y>
          <width>501</width>
          <height>161</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Franklin Gothic Demi Cond</family>
          <pointsize>32</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 20px;
border-width: 2px;
border-color: rgb(0, 0, 0);
border-style: solid;
background: transparent
</string>
        </property>
        <property name="text">
         <string>Quit</string>
        </property>
       </widget>
       <widget class="QLabel" name="GameTitle">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>830</y>
          <width>971</width>
          <height>191</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Franklin Gothic Demi</family>
          <pointsize>110</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: none;
font-color: black;</string>
        </property>
        <property name="text">
         <string>Project Zen</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>0</y>
          <width>901</width>
          <height>821</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(../MyResources/YinYang.png)</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Load">
       <widget class="QPushButton" name="MainMenuLoad">
        <property name="geometry">
         <rect>
          <x>1280</x>
          <y>700</y>
          <width>501</width>
          <height>161</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Franklin Gothic Demi Cond</family>
          <pointsize>32</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 20px;
border-width: 2px;
border-color: rgb(0, 0, 0);
border-style: solid;
background: transparent
</string>
        </property>
        <property name="text">
         <string>Main Menu</string>
        </property>
       </widget>
       <widget class="QFrame" name="frame_2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>50</y>
          <width>921</width>
          <height>961</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-position: center;
background-repeat: no-repeat;
background-image: url(../MyResources/loadbackground.png)
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QTreeView" name="saveGameTree">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>80</y>
           <width>771</width>
           <height>821</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Franklin Gothic Demi</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideMiddle</enum>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="headerDefaultSectionSize">
          <number>200</number>
         </attribute>
         <attribute name="headerHighlightSections">
          <bool>true</bool>
         </attribute>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_3">
        <property name="geometry">
         <rect>
          <x>1270</x>
          <y>110</y>
          <width>511</width>
          <height>461</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(../MyResources/tree.png)</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="End">
       <widget class="QPushButton" name="NewGame2">
        <property name="geometry">
         <rect>
          <x>1360</x>
          <y>160</y>
          <width>501</width>
          <height>161</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Franklin Gothic Demi Cond</family>
          <pointsize>32</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 20px;
border-width: 2px;
border-color: rgb(0, 0, 0);
border-style: solid;
background: transparent
</string>
        </property>
        <property name="text">
         <string>New Game</string>
        </property>
       </widget>
       <widget class="QPushButton" name="loadGame2">
        <property name="geometry">
         <rect>
          <x>1360</x>
          <y>410</y>
          <width>501</width>
          <height>161</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Franklin Gothic Demi Cond</family>
          <pointsize>32</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 20px;
border-width: 2px;
border-color: rgb(0, 0, 0);
border-style: solid;
background: transparent
</string>
        </property>
        <property name="text">
         <string>Load Game</string>
        </property>
       </widget>
       <widget class="QPushButton" name="MainMenuEnd">
        <property name="geometry">
         <rect>
          <x>1360</x>
          <y>660</y>
          <width>501</width>
          <height>161</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Franklin Gothic Demi Cond</family>
          <pointsize>32</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 20px;
border-width: 2px;
border-color: rgb(0, 0, 0);
border-style: solid;
background: transparent
</string>
        </property>
        <property name="text">
         <string>Main Menu</string>
        </property>
       </widget>
       <widget class="QFrame" name="frame_4">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>20</y>
          <width>1151</width>
          <height>1031</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(../MyResources/sun.png);
background-position: center;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="GameTitle_2">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>-10</y>
           <width>991</width>
           <height>191</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Franklin Gothic Demi</family>
           <pointsize>110</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background: none;
font-color: black;</string>
         </property>
         <property name="text">
          <string>Enlightenment!</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="Game">
       <widget class="QPushButton" name="SaveGame">
        <property name="geometry">
         <rect>
          <x>1320</x>
          <y>110</y>
          <width>501</width>
          <height>161</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Franklin Gothic Demi Cond</family>
          <pointsize>32</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 20px;
border-width: 2px;
border-color: rgb(0, 0, 0);
border-style: solid;
background: transparent
</string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
       <widget class="QPushButton" name="SolveGame">
        <property name="geometry">
         <rect>
          <x>1320</x>
          <y>350</y>
          <width>501</width>
          <height>161</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Franklin Gothic Demi Cond</family>
          <pointsize>32</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 20px;
border-width: 2px;
border-color: rgb(0, 0, 0);
border-style: solid;
background: transparent
</string>
        </property>
        <property name="text">
         <string>Solve</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Quit">
        <property name="geometry">
         <rect>
          <x>1320</x>
          <y>580</y>
          <width>501</width>
          <height>161</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Franklin Gothic Demi Cond</family>
          <pointsize>32</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 20px;
border-width: 2px;
border-color: rgb(0, 0, 0);
border-style: solid;
background: transparent
</string>
        </property>
        <property name="text">
         <string>Quit</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="playField">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>110</y>
          <width>681</width>
          <height>681</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Franklin Gothic Medium</family>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background: transparent;
border-style: none;
border-top: 4px solid #000000;
border-left: 4px solid #000000;
border-right: 4px solid #000000;
border-bottom: 4px solid #000000;
gridline-color:  #000000;
font-size: 20pt;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="autoScroll">
         <bool>false</bool>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <property name="rowCount">
         <number>9</number>
        </property>
        <property name="columnCount">
         <number>9</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>75</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>75</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>75</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>75</number>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
       </widget>
       <widget class="QFrame" name="frame_5">
        <property name="geometry">
         <rect>
          <x>1100</x>
          <y>100</y>
          <width>781</width>
          <height>741</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(../MyResources/leaf.png)</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QFrame" name="FrameBarH2">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>562</y>
          <width>681</width>
          <height>4</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QFrame" name="frameBarH1">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>336</y>
          <width>681</width>
          <height>4</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QFrame" name="FrameBarV1">
        <property name="geometry">
         <rect>
          <x>722</x>
          <y>110</y>
          <width>4</width>
          <height>681</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QFrame" name="FrameBarV2">
        <property name="geometry">
         <rect>
          <x>494</x>
          <y>110</y>
          <width>4</width>
          <height>681</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="VictoryButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>1320</x>
          <y>810</y>
          <width>501</width>
          <height>161</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Franklin Gothic Demi Cond</family>
          <pointsize>32</pointsize>
         </font>
        </property>
        <property name="toolTipDuration">
         <number>0</number>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 20px;
border-width: 2px;
border-color: rgb(0, 0, 0);
border-style: solid;
background: transparent</string>
        </property>
        <property name="text">
         <string>Victory!</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="fileSaveName">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>420</y>
          <width>511</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Franklin Gothic Demi Cond&quot;;
text-align: center;
background-color: black;
border-radius: 20px;
border-width: 2px;
border-color: rgb(0, 0, 0);
border-style: solid;
color:white;</string>
        </property>
        <property name="text">
         <string>Enter File Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <zorder>FrameBarH2</zorder>
       <zorder>frameBarH1</zorder>
       <zorder>FrameBarV1</zorder>
       <zorder>FrameBarV2</zorder>
       <zorder>frame_5</zorder>
       <zorder>SaveGame</zorder>
       <zorder>SolveGame</zorder>
       <zorder>Quit</zorder>
       <zorder>playField</zorder>
       <zorder>VictoryButton</zorder>
       <zorder>fileSaveName</zorder>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
